tools/rtpsrelay/RtpsRelay.cpp

```diff
@@ -266,193 +266,207 @@ int ACE_TMAIN(int argc, ACE_TCHAR* argv[])
   const int crypto = 0;
 #endif
 
-  SpdpHandler spdp_vertical_handler(reactor, relay_addresses, association_table, lifespan, rtps_discovery, application_domain, application_participant_id, crypto, spdp);
-  SedpHandler sedp_vertical_handler(reactor, relay_addresses, association_table, lifespan, rtps_discovery, application_domain, application_participant_id, crypto, sedp);
-  DataHandler data_vertical_handler(reactor, relay_addresses, association_table, lifespan, rtps_discovery, application_domain, application_participant_id, crypto);
+  DDS::Subscriber_var bit_subscriber = application_participant->get_builtin_subscriber();
 
-  spdp_horizontal_handler.vertical_handler(&spdp_vertical_handler);
-  sedp_horizontal_handler.vertical_handler(&sedp_vertical_handler);
-  data_horizontal_handler.vertical_handler(&data_vertical_handler);
+  GuidRelayAddressesTypeSupportImpl::_var_type guid_relay_addresses_ts =
```

Comment:
```suggestion
  GuidRelayAddressesTypeSupport_var guid_relay_addresses_ts =
```
Not a big deal but I'd rather stick to the standard mapping in this case (also below with WriterEntry, just search for _var_type).